hadoop-hdds/client/src/main/java/org/apache/hadoop/hdds/scm/storage/ECBlockOutputStream.java

```diff
@@ -115,20 +115,43 @@ ContainerCommandResponseProto> executePutBlock(boolean close,
     }
 
     BlockData checksumBlockData = null;
+    boolean foundStripeChecksum = false;
```

Comment:
We can use `checkSumBlockData == null` to check if it was found, rather than adding a new boolean I think.

Reply:
I'm using this boolean to log only when `stripeChecksum` wasn't found, independent of whether `checksumData` was found or not. It's just to indicate that `stripeChecksum` wasn't found in this version, which can be helpful when debugging.
1. Use the boolean to log when stripeChecksum wasn't found in any index.
2. Instead of throwing, log at line 160 if checksumData was also not found (based on your comment below).